__tests__/app.test.js

```diff
@@ -195,4 +195,40 @@ describe('/api/articles/:article_id/comments', () => {
             expect(typeof postedComment.created_at).toBe('string')
         })
     })
-})
```

Comment:
I would recommend adding another test for
Status 201, ignores unnecessary properties